profiles/tests/test_gql_update_my_profile_mutation.py

```diff
@@ -673,6 +708,56 @@ def test_normal_user_can_remove_email(rf, user_gql_client, email_data):
     assert dict(executed["data"]) == expected_data
 
 
+def test_normal_user_can_remove_all_emails(rf, user_gql_client, email_data):
+    profile = ProfileWithPrimaryEmailFactory(user=user_gql_client.user, emails=2)
+    primary_email = profile.emails.filter(primary=True).first()
+    email = profile.emails.filter(primary=False).first()
+    request = rf.post("/graphql")
+    request.user = user_gql_client.user
+
+    t = Template(
+        """
+            mutation {
+                updateMyProfile(
+                    input: {
+                        profile: {
+                        removeEmails:[
+                            "${primary_email_id}",
+                            "${email_id}",
+                        ]
+                    }
+                }
+            ) {
+                profile{
+                    emails{
+                        edges{
+                        node{
+                            id
+                            email
+                            emailType
+                            primary
+                        }
+                        }
+                    }
+                }
+            }
+            }
+        """
+    )
+
+    expected_data = {"updateMyProfile": {"profile": {"emails": {"edges": []}}}}
+
+    mutation = t.substitute(
+        primary_email_id=to_global_id(type="EmailNode", id=primary_email.id),
+        email_id=to_global_id(type="EmailNode", id=email.id),
+        email=email_data["email"],
+        email_type=email_data["email_type"],
+        primary=str(email_data["primary"]).lower(),
```

Comment:
The template doesn't contain these three placeholders. After removing these, the `email_data` fixture isn't needed in this test anymore.